treeherder/extract/README.md

```diff
@@ -0,0 +1,41 @@
+
+# Extracting to BigQuery
+
+## Run in Heroku
+
+
+### Set Environment variables
+
+The Heroku ***Config Vars*** are not delivered to Python environment variables; there is an obfuscation step which changes the characters as shown. *The JSON String is the JSON encoding of the realized string, for clarity.*
+
+**Some escaping examples**
+
+| Heroku Config Var |  Realized   | JSON String*         |
+| ----------------- | ----------- | -------------------- |
```

Comment:
What do you mean by the term "Realized"?
Why is there an asterisk after "JSON String"?

Reply:
The "Realized" is what you will observe in the environment variable.  The star is in reference to the note below (clarity regarding what is realized)